commonmark-ext-replacement/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'java'
 
 dependencies {
-    compile 'com.atlassian.commonmark:commonmark:0.10.0'
+    implementation 'com.atlassian.commonmark:commonmark:0.10.0'
 
     testCompile 'junit:junit:4.12'
```

Comment:
Could we also change it to `testImplementation`?

Reply:
Updated 👍